djgeojson/serializers.py

```diff
@@ -363,14 +370,21 @@ def serialize(self, queryset, **options):
 
         self.start_serialization()
 
-        if isinstance(queryset, ValuesQuerySet):
-            self.serialize_values_queryset(queryset)
+        if django.VERSION >= (1, 9):
+            if isinstance(queryset, list):
+                self.serialize_object_list(queryset)
+
+            elif isinstance(queryset, QuerySet):
+                self.serialize_queryset(queryset)
+        else:
+            if isinstance(queryset, ValuesQuerySet):
+                self.serialize_values_queryset(queryset)
 
-        elif isinstance(queryset, list):
-            self.serialize_object_list(queryset)
+            elif isinstance(queryset, list):
+                self.serialize_object_list(queryset)
```

Comment:
By the way, you don't need to duplicate these. The only one that is conditional is the ValuesQuerySet one.